2024-11-04 15:30:59,882 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters']
2024-11-04 15:30:59,893 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters', 'update']
2024-11-04 15:30:59,931 DEBUG    root            Running [gcloud.container.clusters.update] with arguments: [--enable-autoscaling: "True", --max-nodes: "5", --min-nodes: "1", NAME: "scaling-demo"]
2024-11-04 15:30:59,932 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): container.googleapis.com:443
2024-11-04 15:31:00,192 DEBUG    urllib3.connectionpool https://container.googleapis.com:443 "GET /v1/projects/proyectotest-1/locations/europe-west1-b/clusters/scaling-demo?alt=json HTTP/1.1" 403 None
2024-11-04 15:31:00,195 WARNING  root            Problem loading details of cluster to update:

ResponseError: code=403, message=Kubernetes Engine API has not been used in project proyectotest-1 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as jdiazderabago@gmail.com which is the active account specified by the [core/account] property.

You can still attempt updates to the cluster.

2024-11-04 15:31:00,198 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): container.googleapis.com:443
2024-11-04 15:31:00,501 DEBUG    urllib3.connectionpool https://container.googleapis.com:443 "PUT /v1/projects/proyectotest-1/locations/europe-west1-b/clusters/scaling-demo?alt=json HTTP/1.1" 403 None
2024-11-04 15:31:00,505 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2024-11-04 15:31:00,538 DEBUG    root            Chosen display Format:default
2024-11-04 15:31:00,540 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2024-11-04 15:31:00,540 DEBUG    root            Chosen display Format:default
2024-11-04 15:31:00,541 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2024-11-04 15:31:00,541 DEBUG    root            Chosen display Format:default
2024-11-04 15:31:00,542 DEBUG    root            (gcloud.container.clusters.update) PERMISSION_DENIED: Kubernetes Engine API has not been used in project proyectotest-1 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as jdiazderabago@gmail.com which is the active account specified by the [core/account] property.
Google developers console API activation
https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  domain: googleapis.com
  metadata:
    consumer: projects/proyectotest-1
    service: container.googleapis.com
  reason: SERVICE_DISABLED
Traceback (most recent call last):
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 842, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/surface/container/clusters/update.py", line 900, in Run
    op_ref = adapter.UpdateCluster(cluster_ref, options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/api_lib/container/api_adapter.py", line 4518, in UpdateCluster
    op = self.client.projects_locations_clusters.Update(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/container/v1/container_v1_client.py", line 979, in Update
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 747, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 753, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 612, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpForbiddenError: HttpError accessing <https://container.googleapis.com/v1/projects/proyectotest-1/locations/europe-west1-b/clusters/scaling-demo?alt=json>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Mon, 04 Nov 2024 14:31:00 GMT', 'server': 'ESF', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'transfer-encoding': 'chunked', 'status': 403}>, content <{
  "error": {
    "code": 403,
    "message": "Kubernetes Engine API has not been used in project proyectotest-1 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.",
    "status": "PERMISSION_DENIED",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.Help",
        "links": [
          {
            "description": "Google developers console API activation",
            "url": "https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1"
          }
        ]
      },
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "SERVICE_DISABLED",
        "domain": "googleapis.com",
        "metadata": {
          "service": "container.googleapis.com",
          "consumer": "projects/proyectotest-1"
        }
      }
    ]
  }
}
>
2024-11-04 15:31:00,544 ERROR    root            (gcloud.container.clusters.update) PERMISSION_DENIED: Kubernetes Engine API has not been used in project proyectotest-1 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as jdiazderabago@gmail.com which is the active account specified by the [core/account] property.
Google developers console API activation
https://console.developers.google.com/apis/api/container.googleapis.com/overview?project=proyectotest-1
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  domain: googleapis.com
  metadata:
    consumer: projects/proyectotest-1
    service: container.googleapis.com
  reason: SERVICE_DISABLED
